Src/coffeescript/cql-execution/test/elm/comparison/test.coffee

```diff
@@ -93,6 +93,14 @@ describe 'Less', ->
   it 'should be false for 5 m < 5 km', ->
     @aLtB_Quantity_diff.exec(@ctx).should.be.true()
 
+  it 'should be false for 5 Cel < 4 m', ->
```

Comment:
The description of this test, and the others, needs to be updated to say "should be **null**..." rather than "should be **false**..."

Reply:
Whoops. I totally screwed that up. I'll fix that.